### PR TITLE
Provide development machine configuration documentation

### PR DESCRIPTION
**Pull request by peterbillen**
**Is your feature request related to a problem? Please describe.**
Provide documentation on how to set up a development machine for contributing

**Describe the solution you'd like**
Additional page in the documents section detailing how to set up the development environment and build the solution

**Describe alternatives you've considered**
NA

**Additional context**
NA

ID
4acc418

### CODE REVIEW

File: docs/pages/contribute/development.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Optionally, install any additional tools that facilitate your work (text editors, ...)
+
+## Get the git repository
+
+Next, clone the git repository for [iesi](https://github.com/metadew/iesi).
+
+## Import external maven dependencies
+
+There are some external libraries used that are not part of maven central. These need to be imported into your local maven maven repository before continuing. From the shell inside IntelliJ, perform the instructions listed in the following [page](http://maven.apache.org/guides/mini/guide-3rd-party-jars-local.html).
```
Comment (BerrevoetsRobbe) [inline docs/pages/contribute/development.md:20]:
```suggestion
There are some external libraries used that are not part of maven central. These need to be imported into your local maven repository before continuing. From the shell inside IntelliJ, perform the instructions listed in the following [page](http://maven.apache.org/guides/mini/guide-3rd-party-jars-local.html).
```

File: docs/pages/contribute/development.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Build using maven goal `clean install project-info-reports:dependencies` and include the profile `dependencies`
+
+### iesi-test
+
+Build using maven goal `clean install`
+
+### iesi-rest-without-microservices
+
+Build using maven goal `clean package project-info-reports:dependencies`
```
Comment (BerrevoetsRobbe) [inline docs/pages/contribute/development.md:43]:
```suggestion
Build using maven goal `clean install project-info-reports:dependencies`
```

File: docs/pages/contribute/development.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{% include navigation.html %}
+
+# How to setup a development machine
+
+## Install development software
+
+First, install the necessary development software on your machine:
+* IntelliJ > [link](https://www.jetbrains.com/idea)
+* IntelliJ plugins: Lomkok > [link](https://plugins.jetbrains.com/plugin/6317-lombok)
+* Hub - command line tool for git > [link](https://github.com/github/hub)
```
Comment (BerrevoetsRobbe) [inline docs/pages/contribute/development.md:10]: Hub is only needed for actual contribution not for creating an IESI instance. Maybe move this software to a dedicated section in pages/contribute/development/contribute.md

Comment (peterbillen) [reply docs/pages/contribute/development.md]: Agreed. Remove the sentence for now. To be added in resolution for #121

Comment (peterbillen) [reply docs/pages/contribute/development.md]: Resolved using commit https://github.com/metadew/iesi/pull/119/commits/cc259ee8e39120af8d92c1763b0b1ff064621e23

**Pull request merged**